_data/simple-icons.json

```
@@ -4852,6 +4852,11 @@
                 ]
             }
         },
+        {
+            "title": "Gsmarena",
+            "hex": "F3F3F3",
```

```suggestion
            "hex": "D50000",
```
This suggestion is open to interpretation. The orange seems to be their hover state on the logo and buttons.